hist/hist/inc/TH2.h

```diff
@@ -70,6 +70,10 @@ class TH2 : public TH1 {
 
 public:
    ~TH2() override;
+   virtual void     AddBinContent(Int_t bin) override;
+   virtual void     AddBinContent(Int_t bin, Double_t w) override;
```

Comment:
`virtual` and `override` should not go in the same declaration

Reply:
I thought that this applied? "When declaring a virtual function in a derived class, that overrides a virtual function in the parent class, you must use both the virtual and the override keywords."
If not, which of the should be used?

Reply:
I'm not sure where that statement comes from, but the latest cpp core guidelines will probably clarify the doubt

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final

Reply:
Ok, I can fix that.

Side note: there are about 100 virtual overrides in the ROOT codebase right now:
`grep -r ' override' | grep 'virtual ' | wc -l`